cmake/config.cmake

```diff
@@ -14,6 +14,24 @@
 
 cmake_minimum_required(VERSION 2.8)
 
+string(TOLOWER ${CMAKE_SYSTEM_NAME} CFG_SYS_NAME)
+string(TOLOWER ${CMAKE_SYSTEM_PROCESSOR} CFG_SYS_PROCESSOR)
+
+if(${CFG_SYS_NAME} STREQUAL "linux")
+  if(${CFG_SYS_PROCESSOR} STREQUAL "x86" OR
+     ${CFG_SYS_PROCESSOR} STREQUAL "x86_64")
+    set(DEVICE_DEPENDS "linux-x86")
+  elseif(${CFG_SYS_PROCESSOR} STREQUAL "armv7l-hf")
+    set(DEVICE_DEPENDS "linux-arm-rpi2")
```

Comment:
What about just use "arm-linux"?
Are there special implementation expected for Raspberry PI 2?